noir/noir-repo/aztec_macros/src/lib.rs

```diff
@@ -1907,7 +1907,7 @@ fn generate_compute_note_hash_and_nullifier_source(note_types: &Vec<String>) ->
                 note_type_id: Field,
                 serialized_note: [Field; 20]
             ) -> pub [Field; 4] {{
-                let note_header = NoteHeader::new(contract_address, nonce, storage_slot);
+                let note_header = dep::aztec::NoteHeader::new(contract_address, nonce, storage_slot);
```

Comment:
If we are doing this in this PR might as well change `note_utils::compute_note_hash_and_nullifier` above to the full path as well